Apps/Examples/Examples/SwiftUI Examples/ClusteringExample.swift

```diff
@@ -0,0 +1,168 @@
+import SwiftUI
+@_spi(Experimental) import MapboxMapsSwiftUI
+
+@available(iOS 14.0, *)
+struct ClusteringExample : View {
```

Comment:
we should probably align the examples across Android and iOS regarding compose/swift UI, let's try to do cross team review on the examples we are going to provide.